uber/models.py

```diff
@@ -630,9 +630,22 @@ def everything(self, location=None):
                                         .order_by(Attendee.full_name).all()
                          if c.AT_THE_CON or not location or int(location) in a.assigned_depts_ints]
             for job in jobs:
-                job._available_staffers = [a for a in attendees if not job.restricted or a.trusted]
+                job._available_staffers = [a for a in attendees if not job.restricted or a.trusted_in(job.location)]
             return jobs, shifts, attendees
 
```

Comment:
Now that I've had to explain this to someone, I wonder if it would be better to include a ``MagModel.prepopulate_cached_property_value`` method.  If you had seen that method getting called then it probably would have been clear in the first place.  I dunno, food for thought.